packages/frontend/components/composed/issue/RelatedIssues/RelatedIssueBlock.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { graphql } from "react-relay";
+import { useMaybeFragment } from "@wdp/lib/api/hooks";
+import * as Styled from "./RelatedIssues.styles";
+import { NamedLink, CoverImage, PrecisionDate } from "components/atomic";
+import { RelatedIssueBlockFragment$key } from "@/relay/RelatedIssueBlockFragment.graphql";
+import { getRouteByEntityType } from "helpers";
+
+export default function RelatedIssueBlock({ data }: Props) {
+  const issue = useMaybeFragment(fragment, data);
+
+  return issue && issue.slug ? (
+    <li key={issue.slug}>
```

Comment:
nit: The `key` only needs to be defined here: https://github.com/NGLPteam/wdp-client/pull/354/files#diff-ec5b51868bbb5499cf5c1cfb4b9c6ec742cfcde2aa4d351700ec9fec461fedb4R23 , and can be removed from this component. 

Since you've been working on this project and OFB, which way do you prefer to structure lists and item components? 
```
# parent component
<ul>
   <li key={}><RelatedIssueBlock /></li>
</ul>
```
or 
```
<ul>
  <RelatedIssueItem key={} />
</ul>
```

I personally find it easier to keep the `li` tag with the `ul` tag, especially during the early stages of development. What do you think?

Reply:
@dananjohnson , I'm curious if you have a preference too ☝️